test/e2e/apimachinery/aggregator.go

```diff
@@ -460,8 +485,26 @@ func TestSampleAPIServer(f *framework.Framework, image string) {
 	if len(unstructuredList.Items) != 0 {
 		framework.Failf("failed to get back the correct deleted flunders list %v from the dynamic client", unstructuredList)
 	}
+}
 
-	cleanTest(client, aggrclient, namespace)
+func testDiscovery(f *framework.Framework) {
+	client := f.ClientSet
+	resources, err := client.Discovery().ServerResourcesForGroupVersion("wardle.k8s.io/v1alpha1")
+	if err != nil {
+		framework.Failf("failed to find the discovery doc for %v: %v", resources, err)
+	}
+	checked := false
+	for _, r := range resources.APIResources {
+		if r.Name == "flunders" {
+			checked = true
+			if r.StorageVersionHash == "" {
```

Comment:
so you just check whether it exists? how about also checking whether its value is expected?

Reply:
Added.